packages/docs/configuration/readme.md

```diff
@@ -155,11 +155,31 @@ module.exports = {
 };
 ```
 
+### Custom container
```

Comment:
**chore :** Move this section to the top to replace the current `### Container` section

Reply:
Done ✅